bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/DigitalInvoiceScreenTests.kt

```diff
@@ -142,24 +140,23 @@ class DigitalInvoiceScreenTests {
         captureScreen.clickFiles()
         pdfUploader.uploadPdfFromFiles("Testrechnung-RA-1.pdf")
         idlingResource.waitForIdle()
-        digitalInvoiceScreen.displayGetStartedButtonOnOnboardingScreen()
+        digitalInvoiceScreen.checkDigitalInvoiceButtonOnOnboardingScreenIsDisplayed()
         digitalInvoiceScreen.clickGetStartedButtonOnOnboardingScreen()
         idlingResource.waitForIdle()
         digitalInvoiceScreen.assertOtherChargesDisplayed()
     }
 
     @Test
-    fun test9_clickProceedButtonOnDigitalInvoiceScreen() {
+    fun test9_checkTransferSummaryButtonIsClickableAfterClickOnProceedButton() {
         mainScreen.clickPhotoPaymentButton()
         onboardingScreen.clickSkipButton()
         captureScreen.clickFilesButton()
         captureScreen.clickFiles()
         pdfUploader.uploadPdfFromFiles("Testrechnung-RA-1.pdf")
         idlingResource.waitForIdle()
-        digitalInvoiceScreen.displayGetStartedButtonOnOnboardingScreen()
+        digitalInvoiceScreen.checkDigitalInvoiceButtonOnOnboardingScreenIsDisplayed()
         digitalInvoiceScreen.clickGetStartedButtonOnOnboardingScreen()
         idlingResource.waitForIdle()
-        digitalInvoiceScreen.assertDigitalInvoiceText()
         digitalInvoiceScreen.clickProceedButton()
         extractionScreen.checkTransferSummaryButtonIsClickable()
```

Comment:
[Request] Please use the result of `checkTransferSummaryButtonIsClickable()` to check if it's true or false here.

Reply:
Assertion added.

```
val isTransferSummaryButtonVisible = extractionScreen.checkTransferSummaryButtonIsClickable()
        assertEquals(true, isTransferSummaryButtonVisible)
```